teuthology/dispatcher/__init__.py

```diff
@@ -7,18 +6,20 @@
 import yaml
 
 from datetime import datetime
+from typing import Dict, List
+
+import teuthology.dispatcher.supervisor as supervisor
+import teuthology.lock.ops as lock_ops
+import teuthology.nuke as nuke
```

Comment:
Again there must a good reason you did this as oppose to ``from teuthology.nuke import nuke``, please explain.

Reply:
You're correct here as well.